code/modules/cm_marines/orbital_cannon.dm

```diff
@@ -384,12 +384,10 @@ var/obj/structure/ship_rail_gun/almayer_rail_gun
 	warhead_kind = "incendiary"
 	icon_state = "ob_warhead_2"
 
+
 /obj/structure/ob_ammo/warhead/incendiary/warhead_impact(turf/target, inaccuracy_amt = 0)
-	var/range_num = max(8 - inaccuracy_amt*2, 3)
-	for(var/turf/TU in range(range_num,target))
-		for(var/obj/flamer_fire/F in TU) // No stacking flames!
-			qdel(F)
-		new/obj/flamer_fire(TU, 10, 50) //super hot flames
+	var/range_num = max(20 - inaccuracy_amt * 2, 15)
```

Comment:
flame_radius() range value is currently clamped at 0-7. You need to remove/modify that line from the proc for this to work.

Reply:
Fixed